docs/api-reference.md

```diff
@@ -9,6 +9,17 @@ If set to a string, it will match based on whether a message's `message.data._pl
 Using `message.data._plugin.type` to match the plugin is a convention we try to push (but not enforce).
 It is therefore also possible to provice an own matching function instead of a string, which will get a message as an argument and should return `true` if this plugin should be used, otherwise `false`.
 
+#### `type: string & rule: () => boolean`
```

Comment:
this does not belong within the "message plugin" documentation. it should be a separate section. 

# Plugins
## Message Plugins
## Input Plugins

Reply:
done